core/src/action.rs

```diff
@@ -150,7 +150,7 @@ pub enum Action {
     /// Last string is the stringified process unique id of this `hdk::get_links` call.
     GetLinks(GetLinksKey),
     GetLinksTimeout(GetLinksKey),
-    RespondGetLinks((FetchMetaData, Vec<Address>)),
+    RespondGetLinks((FetchMetaData, Vec<(Address, CrudStatus)>)),
```

Comment:
@AshantiMutinta Is a tuple future proof enough? Do you imagine other stuff getting associated with address here?

Reply:
I think for now it works because serialization will save space because no field name values. When it grows, should definitely use a struct

Reply:
I think for now it works because serialization will save space because no field name values. When it grows, should definitely use a struct